lib/screens/authentication/recover/recover_account_found/recover_account_found_errors.dart

```diff
@@ -0,0 +1,16 @@
+import 'package:flutter/widgets.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+
+enum RecoverAccountFoundError{ NoGuardians, Unknown}
+
+extension LocalizedRecoverAccountErrors on RecoverAccountFoundError {
+  String localizedDescription(BuildContext context) {
+    final localization = AppLocalizations.of(context)!;
+    switch (this) {
+      case RecoverAccountFoundError.NoGuardians:
+        return localization.recoverAccountFoundMapperNoGuardiansError;
+      case RecoverAccountFoundError.Unknown:
+        return localization.recoverAccountFoundMapperUnknownError;
+    }
+  }
+}
```

Comment:
Using exesions in enums is great, but I think this is going to scale quickly in the number of files to be maintained.

Reply:
@RaulUrtecho Do you mean you are concerned it will NOT scale? In other words, too many files and enum extensions will create a problem?

Reply:
In terms of that we will have more files, although it may not be a problem. not a blocker for this PR.